packages/gateway/deploy/.env

```diff
@@ -0,0 +1,12 @@
+TIMELD_GATEWAY_GATEWAY=
+TIMELD_GATEWAY_GENESIS=true
+TIMELD_GATEWAY_DATA_PATH=/opt/timeld/data
+TIMELD_GATEWAY_AUTH__KEY=timeld.rootkey:
+TIMELD_GATEWAY_SMTP__HOST=
+TIMELD_GATEWAY_SMTP__FROM=
+TIMELD_GATEWAY_SMTP__AUTH__USER=
+TIMELD_GATEWAY_SMTP__AUTH__PASS=
+TIMELD_GATEWAY_ADDRESS__PORT=
+TIMELD_GATEWAY_ADDRESS__HOST=0.0.0.0
+TIMELD_GATEWAY_ADDRESS__PATH=
```

Comment:
I suggest we drop the PATH option for now as we don't know whether it works (if you agree I'll update my jsdoc too).

Reply:
I’m fine with that; I’ll get onto that later this morning.